doc/nrf/ug_nrf5340.rst

```diff
@@ -149,6 +149,12 @@ To run the full Bluetooth LE stack on the network core, the |NCS| provides the :
 
 For the application core, use a compatible Bluetooth LE sample, for example, the :ref:`zephyr:bluetooth-beacon-sample` sample.
 
+Bluetooth mesh
+--------------
+
+Bluetooth mesh operates on Bluetooth Low Energy (LE) and is compatible with core specification version 4.0 and higher.
+For the application core, the |NCS| provides several Bluetooth mesh samples along with the :ref:`Bluetooth Low Energy samples <ble_samples>`.
+In addition, you can find Bluetooth mesh samples with :ref:`Bluetooth samples in Zephyr <zephyr:bluetooth-samples>`.
```

Comment:
Is there a reason why this is shorter than the one for 52 series? I'm sure there is, I'm just curious as I'm not overly familiar with this part of the doc.

Reply:
No particular reason, except that we're a little bit short on time for or the 1.8.0 release. So it's a start, just wanting to at least highlight the Bluetooth mesh in the UG. I would like to see the section get expanded at least a little bit more for the next release, with a nice image similar to one for Thread and Zigbee for example.